src/actors/FilterBarActor.js

```diff
@@ -0,0 +1,88 @@
+import * as SharedUtils from '../utils/SharedUtils';
+
+export class FilterBarActor {
```

Comment:
Besides requiring a singleton "ActionCreator" class in every component that needs to act on something, how would you access the Actions in components?